crates/core/src/commands/prune.rs

```diff
@@ -754,6 +755,8 @@ impl PrunePlan {
     /// # Errors
     ///
     // TODO: add errors!
+    #[allow(clippy::too_many_lines)]
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
actually this lint is right: This method is never returning an error.

Reply:
Yes, it is. Currently. The problem is, I want to remove these things only, when we have taken care about the unwraps/expects. There might be a lot of API breaking changes coming there and maybe we will need a result here anyway. So for now this should be ok.